src/parquet/arrow/schema.cc

```diff
@@ -565,12 +576,12 @@ Status FieldToNode(const std::shared_ptr<Field>& field,
       auto struct_type = std::static_pointer_cast<::arrow::StructType>(field->type());
       return StructToNode(struct_type, field->name(), field->nullable(), properties,
                           arrow_properties, out);
-    } break;
```

Comment:
Accidental deletion?

Reply:
No, the `break` isn't necessary because there's a `return` right?

Reply:
Ah, didn't see that in the quick review.